### DISCUSSION TITLE
Bard text editing issues

### DISCUSSION THREAD

**Discussion by spaceageliving** [statamic/cms#7145] [General]
Some of our writers are experiencing issues when editing text in Bard fields--applying styles to text (like italics) causes the line to break as shown below. Rending the page works fine, as does the in-editor preview mode.

Still trying to collect details from the users; there seems to be a correlation to this behavior when pasting in text from Word.

Has anybody experienced this, and is there a fix?

![Image](https://user-images.githubusercontent.com/874600/204622057-1849bf70-f8ce-4f57-8f80-aa1ba3b8cd2d.jpeg)

**Comment by edalzell** [comment]
Never seen that before, if you can reproduce it on a blank statamic site, please enter an issue.

**Reply by spaceageliving** [reply]
Will do, thanks.
